frontend/src/ThemedApp.tsx

```diff
@@ -4,20 +4,32 @@ import { Global } from "@emotion/core"
 import ThemeProvider from "components/core/ThemeProvider"
 import { LocalStore } from "lib/storageUtils"
 import {
+  AUTO_THEME,
   ThemeConfig,
   getDefaultTheme,
+  getSystemTheme,
   globalStyles,
   lightTheme,
   darkTheme,
 } from "theme"
 import AppWithScreencast from "./App"
 
+const autoTheme = {
+  ...getSystemTheme(),
```

Comment:
Hm, not the biggest deal, but it feels a bit weird to me that for `autoTheme` we copy the `ThemeConfig` into local storage to never use it again / have to worry about what we copied being stale.

Would it be preferable to tweak the types so that what's parsed out of localStorage in `getDefaultTheme` is a `Partial<ThemeConfig>`? Then it can get upgraded to a full-fledged `ThemeConfig` in the case that it's not the auto theme.

Reply:
I like that idea!